logstash-core/lib/logstash/environment.rb

```diff
@@ -20,6 +20,8 @@ module Environment
     Setting::NullableString.new("path.config", nil, false),
  Setting::WritableDirectory.new("path.data", ::File.join(LogStash::Environment::LOGSTASH_HOME, "data")),
     Setting::NullableString.new("config.string", nil, false),
+                    Setting.new("modules.config", Array, []),
```

Comment:
Does `modules.name` make more sense? Or `modules.cli`?  I'm partial to `modules.cli`